DevCycle/DevCycleClient.swift

```diff
@@ -481,22 +479,46 @@ public class DevCycleClient {
             })
     }
 
+    @available(*, deprecated, message: "Use the non-throwing identifyUser method instead")
+    public func identifyUser(user: DevCycleUser, callback: IdentifyCompletedHandler? = nil) throws {
+        guard let currentUser = self.user, !currentUser.userId.isEmpty,
+            !user.userId.isEmpty
+        else {
+            throw ClientError.InvalidUser
+        }
+
+        self._identifyUser(user: user, currentUser: currentUser, callback: callback)
+    }
+
+    public func identifyUser(user: DevCycleUser, callback: @escaping IdentifyCompletedHandler) {
```

Comment:
@jsalaber so I had to create a new method interface (making `callback` non nullable) as you can't have the same method interface where one is `throws` and one isn't. I could also just keep `callback` nullable and rename it to something like `completion`.

Reply:
yeah a rename is fine to keep it nullable